Formula/packs.rb

```diff
@@ -4,7 +4,7 @@
 class Packs < Formula
   desc "A pure Rust implementation of packwerk, a gradual modularization tool for Ruby"
   homepage "https://github.com/alexevanczuk/packs"
-  url "https://github.com/alexevanczuk/packs/releases/download/v0.1.80/packs-mac.tar.gz"
+  url "https://github.com/alexevanczuk/packs/releases/download/v0.1.85/packs-mac.tar.gz"
   sha256 "f7b1dd465b8f79b67dd83e5cf0dc868a24fdc028c8ce219138e1775a8afa7239"
```

Comment:
Can you try installing this package locally? I think that should update the sha256.

Reply:
Whoops. I ended up calculating the new SHA manually. Thanks for catching that!

```bash
joey@gusto:zenpayroll $ curl -L https://github.com/alexevanczuk/packs/releases/download/v0.1.80/packs-mac.tar.gz -o packs.tar.gz
joey@gusto:zenpayroll $ shasum -a 256 packs.tar.gz
f7b1dd465b8f79b67dd83e5cf0dc868a24fdc028c8ce219138e1775a8afa7239  packs.tar.gz

joey@gusto:zenpayroll $ curl -L https://github.com/alexevanczuk/packs/releases/download/v0.1.85/packs-mac.tar.gz -o packs.tar.gz
joey@gusto:zenpayroll $ shasum -a 256 packs.tar.gz
6258fc183c1f71f24007dac6deffe58cbf18d40120ec99b937ce97f000fff50d  packs.tar.gz
```